webpack.config.js

```diff
@@ -23,5 +23,8 @@ module.exports = {
                 }
             }
         ]
+    },
+    node: {
+        fs: 'empty'
```

Comment:
This was added to resolve errors with webpack after adding `skiz-parser` as a dependency.